### PR TITLE
edit profile

### PR DESCRIPTION
**Pull request by ZaurezTimmapuri**
edit profile activity has been completed

### CODE REVIEW

File: application/app/src/main/java/com/example/csi_dmce/Profile.kt [hunk 1]
```diff
@@ -15,23 +19,37 @@ class Profile: AppCompatActivity() {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_profile)
 
-        title_Heading = arrayListOf(
-
-
-
+        val profile_button = findViewById<Button>(R.id.botton_profile)
```
Comment (Crystalsage) [inline application/app/src/main/java/com/example/csi_dmce/Profile.kt]: There's a typo in `botton_profile`.

File: application/app/src/main/java/com/example/csi_dmce/Profile.kt [hunk 2]
```diff
@@ -15,23 +19,37 @@ class Profile: AppCompatActivity() {
...
-        Description = arrayListOf(
 
+        name.setText(name_view)
+        branch.setText(branch_view)
+        roll.setText(roll_view)
+        email.setText(email_view)
+        password.setText(password_view)
 
 
+        title_Heading = arrayListOf(
```
Comment (Crystalsage) [inline application/app/src/main/java/com/example/csi_dmce/Profile.kt]: Can we remove these empty newlines?

File: application/app/src/main/java/com/example/csi_dmce/Profile.kt [hunk 3]
```diff
@@ -15,23 +19,37 @@ class Profile: AppCompatActivity() {
...
 
 
+        )
 
+        Description = arrayListOf(
```
Comment (Crystalsage) [inline application/app/src/main/java/com/example/csi_dmce/Profile.kt]: Empty newlines.

File: application/app/src/main/java/com/example/csi_dmce/Profile.kt [hunk 4]
```diff
@@ -57,4 +75,9 @@ class Profile: AppCompatActivity() {
         newRecyclerView.adapter = ProfileEventsAdapter(newArrayList)
 
     }
+
+
+
+
+
```
Comment (Crystalsage) [inline application/app/src/main/java/com/example/csi_dmce/Profile.kt]: Empty newlines.

**Review by Crystalsage — CHANGES_REQUESTED** [review]
I've left some suggestions. Can you fix them? LGTM otherwise. Thanks for working on this! :)

**Comment by ZaurezTimmapuri** [comment]
changes have been made and has been updated

**Review by Crystalsage — APPROVED** [review]
Thanks for your work. LGTM.

**Pull request merged**